build/rust/Cargo.lock

```diff
@@ -738,7 +738,9 @@ dependencies = [
 
 [[package]]
 name = "either"
-version = "1.6.1"
+version = "1.8.1"
```

Comment:
Please copy deleted upstream `either` to `brave/third_party/rust/...` and update patch path

Reply:
Fixed in afb650a9eff98b9599297835907ee4542136b249. Notice though that we are downgrading it back to `1.6.1`, because that's the version that was in chromium upstream.